all-of-us/workbench
# Chavanr/user admin UI

**Test case for the User Admin UI:**
1. Verify the user-admin table col names
2. Search the user name
3. Click on the Bypass link - Verify the bypass modules for the username
4. Verify the user lockout and status for the username
5. Click on the name link - verify if taken to the user- admin profile page (opens in a new tab)
6. click on the previous tab to view the user-admin table
7. Click on the audit link - verify if taken to the user- audit page (opens in a new tab)


---
**PR checklist**

- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
